src/components/Comments/ViewComments.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import dateFormat from 'dateformat';
+import PropTypes from 'prop-types';
+
+export const ViewComments = ({ allComments, AvatarIcon }) => (
+  <div
```

Comment:
Make this an article tag. It is a stand-alone content

Reply:
All right